src/pages/ProductsPage/ProductsPageComponent.tsx

```diff
@@ -1,19 +1,15 @@
 import { Box } from '@mui/material';
 import Button from '@mui/material/Button';
-import { GridToolbar } from '@mui/x-data-grid/components/toolbar';
-import { DataGrid } from '@mui/x-data-grid/DataGrid';
-import { productTableColumns } from '../../constants/productTableColumns';
-import { ProductModel } from '../../models/ProductModel';
+import ProductsListContainer from './ProductsListContainer';
 
 interface Props {
   onGetProductsClick: () => void;
-  products: ProductModel[];
 }
 
 export default function ProductsPageComponent(props: Props) {
   return (
     <>
-      <Box textAlign="center" marginBottom="8px">
+      <Box marginBottom="8px">
```

Comment:
use sx={{marginBottom="8px"}}

Reply:
The appropriate logic has been implemented into the sx prop